tests/src/test/scala/common/Wsk.scala

```diff
@@ -1040,13 +1040,14 @@ trait RunWskCmd extends Matchers {
             env: Map[String, String] = Map("WSK_CONFIG_FILE" -> ""),
             workingDir: File = new File("."),
             stdinFile: Option[File] = None,
-            showCmd: Boolean = false): RunResult = {
+            showCmd: Boolean = false,
+            hideFromOutput: Seq[String] = Seq()): RunResult = {
         val args = baseCommand
         if (verbose) args += "--verbose"
-        if (showCmd) println(args.mkString(" ") + " " + params.mkString(" "))
+        if (showCmd) println(hideStr(args.mkString(" ") + " " + params.mkString(" "), hideFromOutput))
```

Comment:
hmm show cmd to me is used for debugging - if i'm debugging, dont think i want anything hidden.
do we actually turn this flag on in our tests?